libobs/obs-display.c

```diff
@@ -235,6 +235,7 @@ static inline bool render_display_begin(struct obs_display *display,
 static inline void render_display_end()
 {
 	gs_end_scene();
+	gs_load_swapchain(NULL);
```

Comment:
Philosophical debate. While it makes it less likely for the display to be rendered to by accident, the code also doesn't do anything for correct usage. I'd rather not, but it's up to Lain.

Reply:
original state of the swap is null， so render end should be null